src/frontend/sharedComponents/icons/Circle.js

```diff
@@ -4,16 +4,26 @@ import { StyleSheet, View } from "react-native";
 import type { ViewStyleProp } from "../../types";
 
 type CircleProps = {
+  color?: string,
+  borderWidth?: number,
   radius?: number,
   children: React.Node,
   style?: ViewStyleProp,
 };
 
-const Circle = ({ radius = 25, style, children }: CircleProps) => (
+const Circle = ({
+  color = "#EAEAEA",
+  borderWidth = 2,
+  radius = 25,
+  style,
+  children,
+}: CircleProps) => (
   <View
     style={[
       styles.circle,
       {
+        borderWidth: color === "#EAEAEA" ? null : borderWidth,
```

Comment:
May be worth removing the `borderWidth` from `styles.circle` and letting this be the source of truth for setting that value i.e. changing `null` to `1`. Just a thought to potentially make this a little easier to follow

Reply:
sorry, initially l didn't realize that the `styles.circle` sets `borderWidth: 1`, so it seemed like your change was setting the width to `null`, when really it would fallback to using `1` since styles.circle is used. My suggestion was to avoid setting that in two different locations and instead only specifying it in this inline object. maybe be irrelevant after recent commits